aries-test-harness/features/steps/0160-connection.py

```diff
@@ -118,7 +118,7 @@ def step_impl(context, invitee):
 
     resp_json = json.loads(resp_text)
 
-    if not hasattr(context, 'connection_id_dict'):
+    if not hasattr(context, 'connection_id_dict') or not hasattr(context.connection_id_dict, invitee):
```

Comment:
This will replace the connection_id_dict with a fresh one if the invitee is not present, which loses what else might have been there.

Adding a missing invitee dict should be outside, eg:
```
if not hasattr(context, 'connection_id_dict'):
    context.connection_id_dict = {}
context.connection_id_dict.setdefault(invitee, {})
```

Reply:
Agreed, done